.github/workflows/build_and_push_to_ecr.yml

```diff
@@ -5,9 +5,6 @@ on:
   push:
     branches:
       - main
-    paths-ignore:
-      - "**/*.md"  # Ignore Documentation changes
-      - ".github/**"  # Ignore GitHub Actions changes
```

Comment:
Same question as: https://github.com/department-of-veterans-affairs/contention-classification-api/pull/54#discussion_r1974419690